src/components/DropdownArrow.js

```diff
@@ -11,12 +11,20 @@ export const DropdownArrow  = ({ open,handleOpen }) => {
       width: '53px',
       height: '45px',
       color: theme.palette.secondary.dark,
+      [theme.breakpoints.down('xs')]: {
+        width: '20.9px',
+        height: '17.78px',
```

Comment:
Do we need pixel accuracy down to decimal places?

Reply:
I think it was specified in figma. Do you want me to change it